### ISSUE TITLE
STAS-F15: Insufficient Colour Contrast

### ISSUE THREAD

**Issue by WestNorthantsRob** [FutureNorthants/northants-website#71]
**Frontend or Backend**: Frontend / Backend / Both

This was not logged with the other Accessibility Issues

The grey on white on the text that indicate the size of the pdf files on the ‘comments-compliments-and-complaints’ page, and several other pages throughout the site, fails colour contrast requirements with a ratio of 2.7.

It should have a contrast ratio of at least 4.5:1

Page with an example: https://www.westnorthants.gov.uk/your-council/comments-compliments-and-complaints

**Comment by AnthonyTaylor** [comment]
@WestNorthantsRob This has been fixed in dev and will be part of the next deploy

**Comment by AnthonyTaylor** [comment]
@WestNorthantsRob this requires a change to the design of the website, Lawrence has approved it but do I need to get approval from anyone else?

Before:
![image](https://user-images.githubusercontent.com/26439303/128311336-e9eb7422-cbc1-437b-8e61-0c2e12418f3b.png)

After:
![image](https://user-images.githubusercontent.com/26439303/128311371-9efc812e-aba7-48ed-af80-4519888a03ca.png)

**Comment by WestNorthantsRob** [comment]
@AnthonyTaylor  - You might want to run it past Richard Beards but as it is an accessibility issue I doubt he will say no.
